geoAssembler/widgets/pyqt.py

```diff
@@ -42,7 +42,7 @@ def __init__(self, app, run_dir=None, geofile=None, levels=None):
         pg.LabelItem(justify='right')
 
         self.geofile = geofile
-        self.levels = levels or [None, None]
+        self.initial_levels = levels or [0, 10000]
```

Comment:
why did you choose 10k as max? From my experience this number seems rather high.

Reply:
It was fairly arbitrary based on the data you have for `--test` mode. I went on the high side because I thought it might not be obvious that you can increase the maximum above its starting point.